### ISSUE TITLE
Uncatchable exeption

### ISSUE THREAD

**Issue by Lord-Evil** [ariovistus/pyd#13]
As always, embedded. try{}catch{} around py_stmts(). But no luck. Sigfaults...
Fatal Python error: deletion of interned string failed

**Comment by ariovistus** [comment]
As always, get me some example code, and I can do something about it.

**Comment by Lord-Evil** [comment]
Yup. For now I reversed my code to the working state and trying to go step by step..
